### PR TITLE
Add QSG Cert Renewal Page

### LINKED ISSUE #448: Add Doc for updating CCM Service upon certificate renewal
## What New Or Updated Would You Like To See?

Add a doc for people to guide them through updating their IIS Certificate binding for the CCM website. 

## Why Is It Needed?

We currently give no guidance on what to do with the QSG solution you go to renew your certificate or need to generate a new self-signed one.

## Additional Context?

Related Issue
https://github.com/chocolatey/choco-quickstart-scripts/issues/131
https://github.com/chocolatey/choco-quickstart-scripts/issues/130

### LINKED ISSUE #449: Add doc for updating Nexus configuration upon certificate renewal
## What New Or Updated Would You Like To See?

Wanna provide a doc for QSG users when it comes time for certificate renewal and they need to update their Nexus keystore file.

## Why Is It Needed?

Currently we give no guidance on the process for updating your Java Keystore file with a new certificate for users of the QSG.

## Additional Context?

https://github.com/chocolatey/choco-quickstart-scripts/issues/130

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
Added page on certificate renewal scripts that were added to QSG.
Also put QSG content under its own folder. 

## Motivation and Context
Needed to document the cert renewal scripts that were merged into QSG.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [X] New documentation page added.

## Change Checklist

* [X] Requires a change to menu structure (top or left hand side)/
* [X] Menu structure has been updated

## Related Issues

https://github.com/chocolatey/choco-quickstart-scripts/issues/130
Fixes #448 
Fixes #449

### CODE REVIEW

**Comment by ryanrichter94** [comment]
Added the new cert renewal page as index 40 because I intend to write up a maintaining QSG components page on index 30 that will link to our Nexus and CCM upgrade documentation.

File: input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+Order: 40
+xref: quick-start-guide-cert-renewal
+Title: Certificate Renewal
+Description: Certificate renewal docs for Nexus and Chocolatey Central Management
+RedirectFrom:
+---
+
+This document is meant to serve as a guide for where to look when needing to renew your SSL certificate(s) for the Nexus and Chocolatey Central Management components of quick start guide.
+
+## Set-NexusCert
```
Comment (steviecoaster) [inline input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md:11]:
```suggestion
## Set-NexusCert.ps1
```

File: input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+This script should be on your quick start guide server at `C:\choco-setup\scripts\Set-NexusCert.ps1`. If it isn't, you can download it from the choco-quickstart-scripts project repo on GitHub at https://github.com/chocolatey/choco-quickstart-scripts/blob/main/scripts/Set-NexusCert.ps1.
```
Comment (steviecoaster) [inline input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md:13]:
```suggestion
This script should be saved on your repository server at `C:\choco-setup\scripts\Set-NexusCert.ps1`

You can download this file directly from [here)(https://github.com/chocolatey/choco-quickstart-scripts/blob/main/scripts/Set-NexusCert.ps1), or copy the below code and manually save it.

File: input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+Write-Host -BackgroundColor Black -ForegroundColor DarkGreen "The script has successfully run and the Nexus service is now rebooting for the changes to take effect."
+}
+```
+
+### What does this script do?
+
+- Prompts for Thumbprint value. This is the certificate thumbprint value you wish to associate to your Nexus instance.
+- Optionally you can specify `-Port` with the port number you want Nexus to run over.
+- The script will remove you previous keystore.jks file
```
Comment (steviecoaster) [inline input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md:103]:
```suggestion
- The script will remove your previous keystore.jks file
```

File: input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+```powershell
+.\Set-NexusCert.ps1 -Thumbprint 'Your_Certificate_Thumbprint_Value' -NexusPort 'Port_Number'
+```
+
+```powershell
+.\Set-NexusCert.ps1 -Thumbprint 'Your_Certificate_Thumbprint_Value'
+```
+
+## Set-CCMCert
```
Comment (steviecoaster) [inline input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md:118]:
```suggestion
## Set-CCMCert.ps1
```

File: input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+This script should be on your quick start guide server at `C:\choco-setup\scripts\Set-CCMCert.ps1`. If it isn't, you can download it from the choco-quickstart-scripts project repo on GitHub at https://github.com/chocolatey/choco-quickstart-scripts/blob/main/scripts/Set-CCMCert.ps1.
```
Comment (steviecoaster) [inline input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md:120]: Same verbage as above

File: input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+            if ((Get-Service chocolatey-central-management).Status -ne 'Running') {
+             Write-Warning "Unable to start Chocolatey Central Management service, please start manually in Services.msc"
+            }
+        }
+}
+```
+
+### What does this script do?
+
+- The script will prompt for a $CertificateThumbprint value when run. Please enter the thumbprint value of the certificate you wish to have the IIS endpoint of Chocolatey Central Management run over.
```
Comment (steviecoaster) [inline input/en-us/guides/organizations/quick-start-guide/certificate-renewal.md:191]: Don't call out the variable here, but rather be generic with what it is.

**Review by steviecoaster — CHANGES_REQUESTED** [review]

**Comment by ryanrichter94** [comment]
@steviecoaster Changes you requested should be in latest push.

**Review by steviecoaster — APPROVED** [review]

**Pull request merged**